pkg/lib/machine-change-auth.html

```diff
@@ -1,93 +1,80 @@
 <div class="modal-header">
-    <button type="button" class="close" data-dismiss="modal"
+    <button type="button" class="close"
         aria-hidden="true">&times;</button>
     <h4 class="modal-title" translate="yes">Log in to {{host}}</h4>
 </div>
 
 <div id="machine-change-auth" class="modal-body">
     {{^available}}
-    <p translate="yes">Cockpit was unable to log into {{#strong}}{{host}}{{/strong}}.</p>
+    <p translate="yes">Unable to log in to
+    {{#strong}}{{full_address}}{{/strong}}. The host does not
```

Comment:
These paragraphs must be a single line in the HTML. Otherwise the string extraction leaves weird/broken whitespace in the original strings; Check `make po/cockpit.pot`:
```
#: pkg/lib/machine-change-auth.html:17
msgid ""
"Unable to log in to    {{#strong}}{{full_address}}{{/strong}} using SSH "
"key    authentication. Please provide the password. You may want to set    "
"up your SSH keys for automatic login."
msgstr ""
```

Reply:
Done.